server/googleActions.js

```diff
@@ -241,6 +264,26 @@ const GoogleActions = {
 		return obj.data.id;
 	},
 
+	checkPermissions : async (auth, brew)=>{
```

Comment:
Following the comment above, I don't see that the output from this `checkPermissions` is used for anything.

Reply:
This is resolved in the latest commit.